Makefile

```
@@ -135,12 +149,42 @@ endif
 
 .PHONY: dev-ui
 dev-ui:
 	cd server/frontend && pnpm dev
 
+.PHONY: dev-seed-users
+dev-seed-users:
+# USER is exported by every POSIX shell, so `ifndef USER` never fires and the
+# target would silently fall back to the current OS username. Require it to be
+# passed on the command line instead.
+ifneq ($(origin USER),command line)
+	$(error USER is required. Usage: make dev-seed-users USER=demo PASS=demo123)
```

checkmake (0.3.2)

[warning] Target "$(error USER is required. Usage" defined multiple times (lines 122 and 164).

(uniquetargets)